LastFm/LastFm.m

```diff
@@ -862,7 +861,7 @@ - (NSOperation *)getSessionForUser:(NSString *)username password:(NSString *)pas
 
     return [self performApiCallForMethod:@"auth.getMobileSession"
                                 useCache:NO
-                              withParams:@{ @"username": [username lowercaseString], @"authToken": authToken }
+                              withParams:@{ @"username": [username lowercaseString], @"password": password }
```

Comment:
That's how it's specified by the Last.FM API documentation. I'm not expecting them to change it any time soon regarding to how long it took them to even make the API working. Still better than the doc they provide. 🤕